migration/migration.go

```diff
@@ -78,6 +79,9 @@ func MigrateIfNeeded(targetDB db.DB, network utils.Network) error {
 	}
 
 	for i := version; i < uint64(len(migrations)); i++ {
+		if log != nil {
+			log.Infow("applying database migration", "stage", fmt.Sprintf("%d/%d", i+1, uint64(len(migrations))))
```

Comment:
```suggestion
			log.Infow("applying database migration", "stage", fmt.Sprintf("%d/%d", i+1, len(migrations)))
```

why do you need the casting here?

Reply:
Leftover from the previous commit, my bad.